CHANGES.rst

```diff
@@ -6,6 +6,12 @@ Release 0.9.0
 
 Loïc Estève
 
+    Joblib hashing now uses the default pickle protocol (2 for Python
+    2 and 3 for Python 3). This makes it very unlikely to get the same
+    hash for a given object under Python 2 and Python 3.
```

Comment:
In particular, this means that the output of `joblib.hash` changes when switching from joblib 0.8.4 to 0.9.0 for Python 3 users. We hope that `joblib.hash` is less likely to change for future versions of joblib but this is not officially guaranteed.